payments/mobile-pay/other-features.md

```diff
@@ -16,7 +16,819 @@ sidebar:
+| `transaction`            | `object`     | Y        | The currenct capture object.                                                                                              |
+| └➔&nbsp;`amount`         | `integer`    | Y        | Amount Entered in the lowest momentary units of the selected currency. E.g. `10000` = `100.00 DKK`, `5000` = `50.00 DKK`. |
+| └➔&nbsp;`vatAmount`      | `integer`    | Y        | Amount Entered in the lowest momentary units of the selected currency. E.g. `10000` = `100.00 DKK`, `5000` = `50.00 DKK`. |
+| └➔&nbsp;`description`    | `string`     | Y        | A textual description of the capture transaction.                                                                         |
+| └➔&nbsp;`payeeReference` | `string(50)` | Y        | A unique reference for the capture transaction. See [`payeeReference`][payeeReference] for details.                       |
+
+The `capture` resource contains information about the capture transaction made
+against a mobilepay payment.
+You can return a specific capture transaction by adding the transaction id to
+the `GET` request.
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/mobilepay/payments/{{ page.paymentId }}",
+    "captures": {
+        "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures",
+        "captureList": [
+            {
+                "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures/6f978046-ad7d-4a7e-8ac9-08d617e01a6f",
+                "transaction": {
+                    "id": "/psp/mobilepay/payments/{{ page.paymentId }}/transactions/6f978046-ad7d-4a7e-8ac9-08d617e01a6f",
+                    "created": "2018-09-11T12:14:20.3155727Z",
+                    "updated": "2018-09-11T12:14:21.3834204Z",
+                    "type": "Capture",
+                    "state": "Completed",
+                    "number": 75100000126,
+                    "amount": 1000,
+                    "vatAmount": 250,
+                    "description": "description for transaction",
+                    "payeeReference": "1234",
+                    "isOperational": false,
+                    "operations": []
+                }
+            }
+        ]
+    }
+}
+```
+
+{:.table .table-striped}
+| Property               | Type     | Description                                                          |
+| :--------------------- | :------- | :------------------------------------------------------------------- |
+| `payment`              | `string` | The relative URI of the payment this capture transaction belongs to. |
+| `captures`             | `object` | The created capture transaction.                                     |
+| └➔&nbsp;`id`           | `string` | The relative URI of the created capture transaction.                 |
+| └➔&nbsp;`stransaction` | `object` | The object representation of the `transaction` resource.             |
```

Comment:
Please describe the properties of the `transaction` object. I think they are described in the `transaction.md` include file.

```suggestion
| └➔&nbsp;`transaction` | `object` | The object representation of the `transaction` resource.             |
```